crypto/asn1/a_gentm.c

```diff
@@ -157,10 +157,10 @@ int ASN1_GENERALIZEDTIME_print(BIO *bp, const ASN1_GENERALIZEDTIME *tm)
     if (BIO_printf(bp, "%s %2d %02d:%02d:%02d%.*s %d%s",
                    _asn1_mon[M - 1], d, h, m, s, f_len, f, y,
```

Comment:
I'd say no to asn1_locl.h. Because it will probably trigger "unused variable" warning in all files that include asn1_locl.h, but the two in question. And I'd say no to tbl_standard's alike as well. Because _asn1_mon is trivial enough. I would even go as far as suggesting to move declaration inside function body...